src/containers/teleport/components/tabList/TabList.tsx

```diff
@@ -0,0 +1,45 @@
+import { useNavigate } from 'react-router-dom';
+
+import ButtonTeleport from 'src/containers/warp/components/buttonGroup/indexBtn';
```

Comment:
component should not be mixed with containers, please use inside components only component-scope units:
components is independent entity
containers is set of components.